R/commonAnovaBayesian.R

```diff
@@ -1118,7 +1118,8 @@
 
     pairs <- utils::combn(variable.levels, 2)
 
-    allSplits <- split(dataset[[.v(dependent)]], dataset[[.v(posthoc.var)]])
+    splitname <- if (model[["analysisType"]] == "RM-ANOVA") .BANOVAdependentName else dependent
```

Comment:
`.v(dependent)`?

Reply:
well, yes it should be. But automatic encoding is on now right?